.perlcriticrc

```diff
@@ -44,6 +44,7 @@ allow=refs
 [TooMuchCode::ProhibitUnusedImport]
 [TooMuchCode::ProhibitUnusedInclude]
 
+[ValuesAndExpressions::ProhibitConstantPragma]
```

Comment:
Re: the commit message, I don't see any uses of `REPLICATION_ACCESS_URI` that would benefit from compiler optimization.

Reply:
While changing this one might be fine, changing the other `RT_*` constants in Replication.pm would break existing DBDefs files that reference them, so I think it's best to leave the file alone for now.